src/components/ha-sidebar.html

```diff
@@ -262,13 +266,13 @@
     if (checks) {
       this.selectPanel(attr);
     }
-  },
+  }
 
-  toggleMenu: function () {
-    this.fire('hass-close-menu');
-  },
+  toggleMenu() {
+    this.dispatchEvent(new CustomEvent('hass-close-menu', {bubbles: true, composed: true}));
```

Comment:
I really liked this helper function, I wonder if we should create a class mixin that adds this method back?